src/styles/annotator/highlights.scss

```diff
@@ -1,35 +1,55 @@
 :root {
   // Default highlight styling configuration
   --hypothesis-highlight-color: rgba(255, 255, 60, 0.4);
+  --hypothesis-highlight-secondColor: rgba(206, 206, 60, 0.4);
+  --hypothesis-highlight-thirdColor: transparent;
+
   --hypothesis-highlight-focused-color: rgba(156, 230, 255, 0.5);
-  --hypothesis-highlight-blend-mode: normal;
-  --hypothesis-highlight-decoration: none;
   --hypothesis-highlight-text-color: inherit;
 
-  --hypothesis-highlight-second-color: rgba(206, 206, 60, 0.4);
-  --hypothesis-highlight-third-color: transparent;
+  --hypothesis-color-cyan: #cffafe;
 
   // Colors available for clustered highlights
-  --hypothesis-color-blue: #e0f2fe;
-  --hypothesis-color-yellow: #fef9c3;
-  --hypothesis-color-purple: #ede9fe;
-  --hypothesis-color-orange: #ffedd5;
-  --hypothesis-color-green: #d1fae5;
+
   --hypothesis-color-grey: #f5f5f4;
-  --hypothesis-color-pink: #ffe4e6;
+  --hypothesis-color-grey-1: #e7e5e4;
+  --hypothesis-color-grey-2: #d6d3d1;
+
+  --hypothesis-color-yellow: rgb(254, 249, 195);
+  --hypothesis-color-yellow-1: rgb(253, 243, 149);
+  --hypothesis-color-yellow-2: rgb(252, 235, 111);
```

Comment:
How were these different colors derived? Is it something that could be expressed easily with a different color system (HSL) or a CSS or SASS function?

Reply:
This is important! I don't want to get too sucked in just now, but I do need to write some things down, somewhere. There is an underlying system, but also some human adjustments. In brief:

* The available base colors are the "100" intensities of several [Tailwind-curated colors](https://tailwindcss.com/docs/customizing-colors). Most of these colors are "two steps" away from the next color, i.e. we use "sky", "violet" and "pink".
* _Most_ of the second colors are the equivalent of multiplying the base color, then at 80% opacity on white (it is certainly possible to derive that in a single arithmetic operation on the RGB channels if we wanted to).
* _Most_ of the third colors are the equivalent of multiplying the second color, at 40% opacity on white.
* A few colors deviate from this based on my own visual perception. Yellow and grey IIRC.
* We're not offering a blue, though I'd like to, because we need _some_ "reserved band" of contrasting color to use for focused/hovered highlights.

I am not proposing this to be the palette we necessarily go with. There's more focused work to do here.

Reply:
If we end up finding that it works better visually to hand-tune the colors, that's OK. In that case I suggest we simply document how we arrived at them, in case we decide to extend the system in future.